packages/eds-core-react/src/components/Typography/Typography.docs.mdx

```diff
@@ -0,0 +1,111 @@
+import { Links, PropsTable, Story, Stack, TypographyGroups } from './../../../.storybook/components'
+import { Typography } from '.'
+
+# Typography
+
+Presents hierarchy and organises information as clearly and efficiently as possible, <br />
+therefore text sizes and styles were developed to balance content density and reading comfort.
+
+<Links 
+    figmaUrl="https://www.figma.com/file/0TbIXrrObWj80Cf7KucKYFL0/Design-Tokens?node-id=2%3A4"
+    documentationUrl="https://eds.equinor.com/0b0c666ab/p/721544-typography/b/32c137"
+    sourceUrl="https://github.com/equinor/design-system/blob/develop/packages/eds-core-react/src/components/Typography/Typography.tsx"
+    npmUrl="https://www.npmjs.com/package/@equinor/eds-core-react"
+></Links>
+
+<Story id="typography-typography--introduction" />
+<PropsTable />
+
+## Usage
+
+**Typeface**
+
+The `Equinor` typeface is not included in `EDS` by default. It is available from the EDS CDN.
+
+<ul>
+  <li>All the fonts:</li>
+</ul>
+
+```tsx
+<link rel="stylesheet" href="https://eds-static.equinor.com/font/equinor-font.css" />
+```
+
+<ul>
+  <li>Individual font:</li>
+</ul>
+
+```tsx
+<link rel="stylesheet" href="https://eds-static.equinor.com/font/equinor-regular.css" />
+```
+
+The `Equinor` typeface is the primary typeface and is available in four weights:
+
+<ul>
+  <li><code>light</code>,</li>
+  <li><code>regular</code>,</li>
+  <li><code>medium</code>,</li>
+  <li><code>bold</code>,</li>
+  <li>with accompanying <code>italics</code>.</li>
+</ul>
+
+Please do not use <code>light</code> in digital interfaces except in special cases where the font size is over <code>48px</code>.
+
+**Paragraph**
+
+<ul>
+  <li>Paragraph length is the number of characters in a line of text, this includes spaces.</li>
+  <li>To ensure readability, line length should be 55-80 characters.</li>
+</ul>
+
+**Component**
+
+```tsx
+import { Typography } from '@equinor/eds-core-react'
+
+<Typography>Sample text</Typography>
+```
+
+## Overview
+
+Typography groups with included varints:  
+
+Example: `<Typography group="input" variant="label"></Typography>`. <br />
+Variants under `heading` and `paragraph` don’t require the group parameter.
+
+<Story id="typography-typography--groups-overview" />
+
+## Examples
+
+### Colors
+
+<Story id="typography-typography--colors" />
+
+### Token property
+
+The `token` property allows manipulate the appearance of the typography. <br />
+Properties: 
+
+<ul>
+  <li><code>color</code>,</li>
+  <li><code>fontFamily</code>,</li>
+  <li><code>fontSize</code>,</li>
+  <li><code>fontWeight</code>,</li>
+  <li><code>fontStyle</code>,</li>
+  <li><code>fontFeature</code>,</li>
+  <li><code>lineHeight</code>,</li>
+  <li><code>textAlign</code>,</li>
+  <li><code>textTransform</code>,</li>
+  <li><code>textDecoration</code>,</li>
+  <li><code>letterSpacing</code>.</li>
+</ul>
```

Comment:
Good suggestion! Maybe make this example closer to how it would look in code
For example:

```javascript
const myToken = {
    color,
    fontFamily,
    fontSize,
    fontWeight,
    lineHeight,
    textAlign,
    textTransform,
    textDecoration,
    letterSpacing,
}
```

Reply:
done